app/views/carts/show.html.erb

```diff
@@ -48,15 +49,19 @@
       </div>
     </div>
   <% end %>
-</div> <br>
+</div>
+<br>
 <div class="text-center">
   <h4>Your total bill is: <%= total_bill %> Rs </h4>
-</div> <br>
+</div>
+<br>
 <div class="d-flex justify-content-center align-items-center mt-3">
   <%= button_to 'Remove All Items', user_cart_path(current_user, cart_id: @cart.id), method: :delete, class: "btn btn-danger", form: { 
     data: { 
-      turbo_confirm: "Are you sure you want remove all items from cart?"
+      turbo_confirm: "Are you sure you want to remove all items from the cart?"
     }
    } %>
+   
+   <%= button_to "Check Out", new_order_path(@cart), method: :get, class: "btn btn-primary" %>
```

Comment:
Please verify do we need to specify method: :get. If not then remove this from here